sound/soc/sof/intel/tgl.c

```diff
@@ -80,6 +80,8 @@ int sof_tgl_ops_init(struct snd_sof_dev *sdev)
 		ipc4_data = sdev->private;
 		ipc4_data->manifest_fw_hdr_offset = SOF_MAN4_FW_HDR_OFFSET;
 
+		ipc4_data->mtrace_type = SOF_IPC4_MTRACE_INTEL_CAVS_2;
```

Comment:
icl.c ?   also CAVS_2

Reply:
@RanderWang, the icl.c did not had the `if (sdev->pdata->ipc_type == SOF_INTEL_IPC4) { ... }` section, so it is not made to work with IPC4. I think it can use the same section as tgl?

Should I add that as a separate patch first?